libutils/file_lib.h

```diff
@@ -97,6 +97,14 @@ int safe_lchown(const char *path, uid_t owner, gid_t group);
 int safe_creat(const char *pathname, mode_t mode);
 
 /**
+ * @brief Sets whether a file descriptor should be inherited by sub processes.
+ * @param fd      File descriptor.
+ * @param inherit Whether to enable inheritance or not.
+ * @return true on success, false otherwise.
+ */
+bool SetInheritFileDescriptor(int fd, bool inherit);
```

Comment:
What's the default if I don't call this function? It's a bit confusing, because it's implied that if I don't call it, the FD is not inherited.